src/multisig.js

```diff
@@ -0,0 +1,184 @@
+const nacl = require('./nacl/naclWrappers');
+const address = require('./encoding/address');
+const encoding = require('./encoding/encoding');
+const txnBuilder = require('./transaction');
+
+/**
+ Utilities for manipulating multisig transaction blobs.
+ */
+
+const ERROR_MULTISIG_MERGE_LESSTHANTWO = new Error("Not enough multisig transactions to merge. Need at least two");
```

Comment:
This is an arbitrary constraint. Although makes sense, I think that threshold of 1 is perfectly reasonable.

Reply:
Oh, this is something else entirely - we need at least two multisig transactions in order to merge them into one.